HiP-FeatureToggle/Models/Entity/FeatureToFeatureGroupMapping.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace PaderbornUniversity.SILab.Hip.FeatureToggle.Models.Entity
+{
+    /// <remarks>
+    /// Since EF Core does not (yet) support many-to-many relationships, the relationship between features
+    /// and feature groups is modelled as two 1-to-many relationships. In other words, this class acts as
+    /// the join table.
+    ///
+    /// [Feature]----------------[FeatureToFeatureGroupMapping]-------------[FeatureGroup]
+    ///          1              n                              m           1
+    /// </remarks>
+    public class FeatureToFeatureGroupMapping
```

Comment:
Of course if this approach is somehow better than the one we used, we can discuss it and maybe do it your way in the future